### PR TITLE
DP-21928-test-wte-playground-git-repo

### PR DESCRIPTION
**Pull request by AliceThompsonNHSE**

### CODE REVIEW

File: scripts/test_python.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('Hello World') #this is a test
```
Comment (AnthonyBaneNHSE) [inline scripts/test_python.py:1]: This is how a review in-line looks and what I was getting at in the call. So I would say something like:

Make sure to remove comments that are not needed.

**Review by AnthonyBaneNHSE — CHANGES_REQUESTED** [review]
Please see the requested change.

**Review by AliceThompsonNHSE — COMMENTED** [review]
Thanks Anthony. Are you saying you would like me to remove the superfluous comment we added?

File: scripts/test_python.py (hunk 1 above)
Comment (AnthonyBaneNHSE) [reply scripts/test_python.py:1]: @AliceThompsonNHSE when you fix the suggested/required changes and let me know (i.e. delete the comment), I can then review the changes. If this changes are correct I will make this as resolved

Comment (AliceThompsonNHSE) [reply scripts/test_python.py]: I just deleted the comment and committed the change. Thanks.

**Comment by AnthonyBaneNHSE** [comment]
Closing the PR as we don't actually want to add this to the repo. Make sure you delete your local and origin branches @AliceThompsonNHSE

**Pull request closed without merging**